### ISSUE TITLE
New Domain

### ISSUE THREAD

**Issue by JasmineCT** [JasmineCT/portfolio#19]
I seem to be having some trouble pointing my portfolio website to my new domain. But GitHub doesn't seem to like it, saying in the master branch The tag `pattern` on line 123 in `/_layouts/branding.html` is not a recognized Liquid tag.
Not sure how to fix this. I couldn't remember how to make it point from my coming soon page to my new domain. @thomasjbradley any ideas?

**Comment by thomasjbradley** [comment]
Your website is hosted on Netfliy, not on GitHub. Because GitHub doesn’t support the pattern library system.

Turn of GitHub Pages hosting in your GitHub settings & then sign into Netlify.com to change your domain.

**Comment by JasmineCT** [comment]
Ohhh okay that makes sense, I completely forgot about Netlify. Everything looks good on hover, and netlify(i changed it yesterday). But it still seems to be pointing to my coming soon page. When I turn off GitHub pages for my coming soon page it gives me a 404 page.

**Comment by thomasjbradley** [comment]
In Hover, where are your Name Servers pointed?
It’s in the left column, near the bottom.

**Comment by JasmineCT** [comment]
I inputted the ones I got from Netlify into the hover name server spots. 
<img width="395" alt="Screen Shot 2020-10-12 at 1 03 22 PM" src="https://user-images.githubusercontent.com/31959259/95772955-8ff33400-0c8b-11eb-9d28-66e1ac4cb9b2.png">
<img width="774" alt="Screen Shot 2020-10-12 at 1 04 05 PM" src="https://user-images.githubusercontent.com/31959259/95772957-91246100-0c8b-11eb-8dde-bb36ce65380a.png">

**Comment by thomasjbradley** [comment]
Okay. And you set up your domain on Netlify?
When did you change the name servers on Hover? Within the last 48 hours?

**Comment by JasmineCT** [comment]
Yup I changed it on netlify yesterday, and I got my new domain a month ago.

**Comment by thomasjbradley** [comment]
Okay—it’s possible we are just running into time problems. Let’s give it another day or two and see if it starts working.

**Comment by JasmineCT** [comment]
Okay sounds great! Thanks so much for your help Thomas SO greatly appreciated!

**Comment by JasmineCT** [comment]
Hmm seems to be giving me this page now. On chrome, safari, firefox. When i click advance it gives me this.
<img width="695" alt="Screen Shot 2020-10-13 at 6 32 09 PM" src="https://user-images.githubusercontent.com/31959259/95923092-bc3dac00-0d82-11eb-8f66-31b08e1ff55d.png">

I went to netlify to check it and its saying this. Not sure if i just click renew because it has my old domain.
<img width="895" alt="Screen Shot 2020-10-13 at 6 34 01 PM" src="https://user-images.githubusercontent.com/31959259/95923127-d37c9980-0d82-11eb-8742-395a6d729316.png">

**Comment by thomasjbradley** [comment]
Inside Hover, the name servers are pointed to Netlify for `jasminegraphicdesigner.ca`?

**Comment by JasmineCT** [comment]
In Netlify I changed the domain to my new one jasminecarlucci.ca, then I took the new DNS' and input them into hover. So they should be pointing to the new domain.
<img width="612" alt="Screen Shot 2020-10-15 at 4 03 24 PM" src="https://user-images.githubusercontent.com/31959259/96180457-70b10c80-0f00-11eb-8455-4d5d6a34080f.png">
<img width="201" alt="Screen Shot 2020-10-15 at 4 06 43 PM" src="https://user-images.githubusercontent.com/31959259/96180465-71e23980-0f00-11eb-9256-6e7b0e90a3e0.png">

**Comment by thomasjbradley** [comment]
The screenshot above, with the SSL error, shows your old domain.

**Comment by JasmineCT** [comment]
Yah I'm not sure why because I changed my domain
<img width="907" alt="Screen Shot 2020-10-16 at 9 33 07 AM" src="https://user-images.githubusercontent.com/31959259/96264885-bd452800-0f92-11eb-9316-6d37f5039193.png">

**Comment by thomasjbradley** [comment]
Maybe try clicking the “Renew certificate” button?

**Comment by JasmineCT** [comment]
Okay I'll try that

**Comment by JasmineCT** [comment]
Oh gosh perfect, It worked!! Thanks so much Thomas! I appreciate your help so very much
